include/GafferScene/RenderManifest.h

```diff
@@ -58,6 +58,10 @@ class GAFFERSCENE_API RenderManifest
 	public :
 		RenderManifest();
 
+		// There probably isn't much good reason to copy a manifest, but we need this for the Python
+		// binding which can't return a shader_ptr
+		RenderManifest( const GafferScene::RenderManifest& other );
```

Comment:
This defeats the point of making `loadFromImageMetadata()` return `const` - someone can copy it and call `acquireID()` to create conflicting IDs. I've pushed https://github.com/GafferHQ/gaffer/pull/6402/commits/be25dfd74db08be1f8f4a4de1dad419c76b2e708 which removes it and adjusts the Python bindings to compensate.